test/setup-fedora-govm.sh

```diff
@@ -54,14 +59,15 @@ EOF
 # For the sake of reproducibility, use fixed versions.
```

Comment:
Comment no longer aligned with code.

Reply:
You mean with this code change?. This change is only touching indentation.

Reply:
Yes, and the change doesn't change the indention of this comment. It should also indent the comment further to the right such that the `#` is in the same column as the code again.

Reply:
oops, I misunderstood the word 'aligned'. I will fix it.

Reply:
Fixed it now.